### ISSUE TITLE
Project folder with hidden components

### ISSUE THREAD

**Issue by tensor5** [Microsoft/TypeScript#9899]
**TypeScript Version:**  2.0.0

**Code**

On Linux if one of the components of the project folder pathname begins with a dot, `tsc` ignores all files in the project:

`.a/projectDir/foo.ts`

`.a/projectDir/tsconfig.json`
```json
{
    "compilerOptions": {
        "outDir": "dist"
    }
}
```

**Expected behavior:**
`.a/projectDir/dist/foo.js`

**Actual behavior:** 
No `dist`

**Comment by mhegazy** [comment]
This is the intended design to avoid automatically loading hidden files and folders. if you need them explicitly you can use `"files"`.

`"include"` should also work with something like `".*"`, but does not seem to be the case.  @riknoll can you take a look at this case?

**Comment by riknoll** [comment]
Should be fixed now that #9528 is merged
